README.md

```diff
@@ -183,6 +183,6 @@ git add manubot/__init__.py
 git commit --message="Set __version__ to $TAG"
 git push
 # Create & push tag (assuming upstream is greenelab remote)
-git tag --annotate $TAG --message="Upgrade to $TAG"
+git tag --annotate $TAG --file=$TAG.md
```

Comment:
Is this reading from the file `release-notes/$TAG.md`?  Does the `release-notes` directory need to be specified?